dialect.go

```diff
@@ -11,6 +11,7 @@ type SQLDialect interface {
 	createVersionTableSQL() string // sql string to create the db version table
 	insertVersionSQL() string      // sql string to insert the initial version table row
 	dbVersionQuery(db *sql.DB) (*sql.Rows, error)
+	lockTableQuery(tx *sql.Tx) error
```

Comment:
Shouldn't we use the same signature as in the SQL functions above?

`lockVersionTableSQL() string`

Reply:
I'm OK with this, otherwise the caller code has to check for empty string and/or error.

Reply:
@1vn ok then. Can you test this after #116 ?

Reply:
This is a breaking API change. We could consider this for v3, but not earlier.